publish.sh

```diff
@@ -13,3 +13,8 @@ git checkout -b $publish_branch
 git push origin $publish_branch
 
 npx lerna publish
+
+# Update package-lock.json files as well
+npx lerna exec npm install
```

Comment:
This is already run in line 5, if the only new line is the commit line up there?

Reply:
- The first `npm install` makes sure everything is up to date, but usually doesn't do anything—if it does, `lerna publish` will fail as the working tree won't be empty
- `lerna publish` updates the version numbers in the package.json files and commits, but doesn't run `npm install` so the lockfiles remain unchanged
- The second `npm install` updates the lockfiles